tidb-in-kubernetes/upgrade-tidb.md

```diff
@@ -0,0 +1,57 @@
+---
+title: 滚动更新 TiDB 集群
+category: how-to
+---
+
+# 滚动更新 TiDB 集群
+
+滚动更新 TiDB 集群时，会按 PD / TiKV / TiDB 的顺序，串行删除 Pod，并创建新版本的 Pod，当新版本的 Pod 正常运行后，再处理下一个 Pod。滚动升级过程会自动处理 PD / TiKV 的 Leader 迁移与 TiDB 的 DDL Owner 迁移。因此，在多节点的部署拓扑下（最小环境：PD * 3、TiKV * 3、TiDB * 2），滚动更新不会影响业务运行。
+
+滚动更新可以用于升级 TiDB 版本，也可以用于更新集群配置。
+
+## 升级 TiDB 版本
+
+1. 修改集群的 `values.yaml` 文件中的 `tidb.image`, `tikv.image`, `pd.image` 的值为新版本镜像；
+2. 执行 `helm upgrade` 命令进行升级：
+
+    ```shell
+    `helm upgrade ${releaseName} charts/tidb-operator -f values.yaml`；
```

Comment:
```suggestion
    helm upgrade ${releaseName} charts/tidb-cluster -f values.yaml
```

Reply:
I will change these to the new helm command form (with repo)